docs/release-notes/currentRelease.md

```diff
@@ -1,60 +1,32 @@
 ---
-title: DevOps Docker Builds, Version 2111 (December 06 2021)
+title: DevOps Docker Builds, Version 2111.1 (December 16 2021)
 ---
-# Version 2111 Release Notes
+# Version 2111.1 Release Notes
 
-## DevOps Docker Builds, Version 2111 (December 06 2021)
+## DevOps Docker Builds, Version 2111.1 (December 16 2021)
 
```

Comment:
I think we need some sort of note that tells customers about the purging of the affected images we are about to do and why we're doing it.  What do you think?

Reply:
This has now been added. Good suggestion 😄